packages/angular/src/components/treemap/treemap.component.ts

```diff
@@ -79,9 +81,13 @@ export class VisTreemapComponent<Datum> implements TreemapConfigInterface<Datum>
   /** Array of accessor functions to defined the nested groups. Default: `[]` */
   @Input() layers: StringAccessor<Datum>[]
 
-  /** A function that accepts a value number and returns a string. Default: `undefined` */
+  /**  */
```

Comment:
Should we add an explanation here?

Reply:
Yeah, I think the wrapper generator doesn't know how to deal with deprecation comments. Don't have time to look into this now, maybe we can just add it manually.